### PR TITLE
Ionic issues

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -95,6 +96,7 @@ import { ConvertCodeToName } from '../pipe/cart/attribute';
         SliderComponent,
         SideMenu,
         ConvertCodeToName,
+        singleDecimalPipe,
```
Comment (reactexcel) [inline src/app/app.module.ts]: change to ReviewDisplayPipe

File: src/components/categoryProducts/categoryProducts.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
                                     <div class="columnForGrid25 Grid50">
                                         <button ion-button icon-left class="btnStar">
                                             <ion-icon name="star"></ion-icon>
-                                            {{feat_prod.data.rating}}
+                                            {{feat_prod.data.rating | singleDecimal}} / 5
```
Comment (reactexcel) [inline src/components/categoryProducts/categoryProducts.html]: do the divide by 5 in pipe/filter itself

File: src/app/app.module.ts (hunk 1 above)
Comment (upexcel) [inline src/app/app.module.ts]: Fixed

File: src/components/categoryProducts/categoryProducts.html (hunk 2 above)
Comment (upexcel) [inline src/components/categoryProducts/categoryProducts.html]: this is not dividing just in html show like 2.3 / 5

**Pull request merged**